README.md

```diff
@@ -1,89 +1,74 @@
-## DeepDetect : Open Source Deep Learning Server & API
+<p align="center"><img src="https://www.deepdetect.com/img/icons/menu/sidebar/deepdetect.svg" alt="DeepDetect Logo" width="60%" /></p>
 
-[![Join the chat at https://gitter.im/beniz/deepdetect](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/beniz/deepdetect?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [![Build Status](https://travis-ci.org/beniz/deepdetect.png)](https://travis-ci.org/jolibrain/deepdetect)
+<h1 align="center"> Open Source Deep Learning Server & API</h1>
+
+[![Join the chat at https://gitter.im/beniz/deepdetect](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/beniz/deepdetect?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
 DeepDetect (http://www.deepdetect.com/) is a machine learning API and server written in C++11. It makes state of the art machine learning easy to work with and integrate into existing applications. It has support for both training and inference, with automatic conversion to embedded platforms with TensorRT (NVidia GPU) and NCNN (ARM CPU).
 
-DeepDetect relies on external machine learning libraries through a very generic and flexible API. At the moment it has support for:
+It implements support for supervised and unsupervised deep learning of images, text, time series and other data, with focus on simplicity and ease of use, test and connection into existing applications. It supports classification, object detection, segmentation, regression, autoencoders, ...
+
```

Comment:
One idea I'm always playing with is having a small screencast (much smaller than the one of https://github.com/PyTorchLightning/pytorch-lightning) that'd show the 3 quick steps (download docker, create service, predict over data). Too early maybe :)

Reply:
Kind of things takes a lot of time to do it right.